### PR TITLE
Tidy review

### PR DESCRIPTION
**Pull request by GalRabin**
**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related PR
https://github.com/demisto/content/pull/9180/

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [x] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/Tidy/IncidentFields/incidentfield-GitHubToken.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+ "associatedToAll": false,
+ "associatedTypes": [
+  "Tidy install"
+ ],
+ "caseInsensitive": false,
+ "cliName": "githubtoken",
```
Comment (bakatzir) [inline Packs/Tidy/IncidentFields/incidentfield-GitHubToken.json:7]: let's add a prefix of tidy

File: Packs/Tidy/IncidentFields/incidentfield-GitHubToken.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ "required": false,
+ "sla": 0,
+ "system": false,
+ "threshold": 72,
+ "type": "shortText",
+ "unmapped": false,
+ "unsearchable": true,
+ "useAsKpi": false,
+ "version": -1,
+ "fromVersion": "5.0.0"
```
Comment (bakatzir) [inline Packs/Tidy/IncidentFields/incidentfield-GitHubToken.json:28]:
```suggestion
 "fromVersion": "6.0.0"
```

File: Packs/Tidy/IncidentTypes/incidenttype-Tidy_install.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+ "locked": false,
+ "name": "Tidy install",
+ "playbookId": "Tidy install content",
+ "readonly": false,
+ "reputationCalc": 0,
+ "system": false,
+ "version": -1,
+ "weeks": 0,
+ "weeksR": 0,
+ "fromVersion": "5.0.0"
```
Comment (bakatzir) [inline Packs/Tidy/IncidentTypes/incidenttype-Tidy_install.json:21]:
```suggestion
 "fromVersion": "6.0.0"
```

File: Packs/Tidy/IncidentFields/incidentfield-GitHubToken.json (hunk 1 above)
Comment (bakatzir) [inline Packs/Tidy/IncidentFields/incidentfield-GitHubToken.json:7]:
```suggestion
 "cliName": "tidygithubtoken",
```

File: Packs/Tidy/IncidentFields/incidentfield-GitHubToken.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  "Tidy install"
+ ],
+ "caseInsensitive": false,
+ "cliName": "githubtoken",
+ "closeForm": false,
+ "content": true,
+ "editForm": true,
+ "group": 0,
+ "hidden": false,
+ "id": "incident_githubtoken",
```
Comment (bakatzir) [inline Packs/Tidy/IncidentFields/incidentfield-GitHubToken.json:13]:
```suggestion
 "id": "incident_tidygithubtoken",
```

File: Packs/Tidy/IncidentFields/incidentfield-GitHubToken.json [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+ "isReadOnly": false,
+ "locked": false,
+ "name": "GitHubToken",
```
Comment (bakatzir) [inline Packs/Tidy/IncidentFields/incidentfield-GitHubToken.json:16]:
```suggestion
 "name": "TidyGitHubToken",
```

File: Packs/Tidy/Playbooks/playbook-Content_developer_setup.yml [hunk 6]
```diff
@@ -0,0 +1,972 @@
...
+  playbookInputQuery:
+  required: true
+  value:
+    simple: ${incident.tidyuser}
+- description: Github token with public key admin permissions.
+  key: Github token
+  playbookInputQuery:
+  required: true
+  value:
+    simple: ${incident.githubtoken}
```
Comment (bakatzir) [inline Packs/Tidy/Playbooks/playbook-Content_developer_setup.yml:18]:
```suggestion
    simple: ${incident.tidygithubtoken}
```

File: Packs/Tidy/Playbooks/playbook-Tidy_install_content.yml [hunk 7]
```diff
@@ -0,0 +1,516 @@
...
+          "y": 410
+        }
+      }
+  "4":
+    conditions:
+    - condition:
+      - - left:
+            iscontext: true
+            value:
+              simple: incident.githubtoken
```
Comment (bakatzir) [inline Packs/Tidy/Playbooks/playbook-Tidy_install_content.yml:165]:
```suggestion
              simple: incident.tidygithubtoken
```

File: Packs/Tidy/Playbooks/playbook-Tidy_install_content.yml [hunk 8]
```diff
@@ -0,0 +1,516 @@
...
+    id: "5"
+    ignoreworker: false
+    nexttasks:
+      '#none#':
+      - "8"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      access_token:
+        simple: ${incident.githubtoken}
```
Comment (bakatzir) [inline Packs/Tidy/Playbooks/playbook-Tidy_install_content.yml:207]:
```suggestion
        simple: ${incident.tidygithubtoken}
```

**Review by bakatzir — APPROVED** [review]
@GalRabin @hod-alpert I've changed the usage of githubtoken to tidygithubtoken in all occurrences, and also the fromversion of all the entities to be from 6.0.0.

Please tend to the failing validations.

**Comment by GalRabin** [comment]
@bakatzir @hod-alpert 
The PR should be forced to merge due to the following errors in validating command:
1.  `galrabin/tidy:1.0`  - Clearly has tag.
```
Packs/Tidy/Integrations/Tidy/Tidy.yml: [DO104] - galrabin/tidy:1.0 - The docker image in your integration/script does not have a tag. Please create or update to an updated versioned image
```

2. Its contribution so it doesn't have to be from demisto/...
```
Packs/Tidy/Integrations/Tidy/Tidy.yml: [DO102] - docker image must be a demisto docker image. When the docker image is ready, please rename it to: demisto/<image>:<tag>
```

3. It exits you can try run the following - `docker pull  galrabin/tidy:1.0`
```
Packs/Tidy/Integrations/Tidy/Tidy.yml: [DO107] - galrabin/tidy:1.0 - Could not find the docker image. Check if it exists in DockerHub: https://hub.docker.com/u/demisto/.
```

From my side, I finished my work on this PR If there aren't additional changes here (:

**Comment by bakatzir** [comment]
We will need t o allow ignoring these errors in the sdk, so it will have to wait to the next release. @anara123 @roysagi 
FYI.

**Pull request merged**